.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
```

Comment:
Do we plan to remove windows permanently?

Reply:
This was causing failures, possibly because it is tracking the latest windows.  I don't see much value in testing it as it causes false positives and I don't believe we've seen a windows-specific issue.  Java is pretty robust cross-platform.  The only thing might be testing the GUI aspects of RapidWright.

Reply:
I have no issue disabling it.